code/parachain/frame/cosmwasm/src/lib.rs

```diff
@@ -208,6 +208,9 @@ pub mod pallet {
 	pub trait Config:
 		frame_system::Config<AccountId = AccountIdOf<Self>> + Send + Sync + Debug
 	{
+		/// Max number of frames a contract is able to push, a.k.a recursive calls.
+		const MAX_FRAMES: u16;
```

Comment:
Why not u8?

Reply:
it was u32 some time ago.

but i guess contract gas stack can be in theory u16, but i am alos pretty sure it will never reach u8.